src/main/java/org/broadinstitute/hellbender/tools/copynumber/coverage/caller/CalledCopyRatioSegment.java

```diff
@@ -0,0 +1,64 @@
+package org.broadinstitute.hellbender.tools.copynumber.coverage.caller;
+
+import org.broadinstitute.hellbender.tools.copynumber.coverage.segmentation.CopyRatioSegment;
+import org.broadinstitute.hellbender.utils.Utils;
+
+public class CalledCopyRatioSegment extends CopyRatioSegment {
+    public enum Call {
+        AMPLIFICATION_CALL("+"),
+        DELETION_CALL("-"),
+        NEUTRAL_CALL("0");
+
+        private final String outputString;
+
+        Call(final String outputString) {  this.outputString = outputString; }
+
+        public String getOutputString() {
+            return outputString;
```

Comment:
Make this a one-liner for consistency.

Reply:
I will make the other method not a one-liner instead.